### PR TITLE
[DATA-1053] Only use `id` as a package identifier in Get/DeletePackageRequest

### PR DESCRIPTION
**Pull request by agreenb**
Corresponding app PR: https://github.com/viamrobotics/app/pull/1151

Context [here](https://docs.google.com/document/d/1kKOhO_PzemaqXoHrP_F3KOYPJ3QMAo7dmNJdugxO5bI/edit?disco=AAAAoWiZ3VE) on the change to the package identifier.

The above app PR moves all usage to `id`. The first usage of PackageService in RDK is already using `id`: https://github.com/viamrobotics/rdk/pull/1829

### CODE REVIEW

**Review by AdamMagaluk — APPROVED** [review]

**Review by stevebriskin — APPROVED** [review]

**Comment by agreenb** [comment]
@stevebriskin Could you force-merge this change? Have the corresponding app change approved and ready to merge right after.

**Pull request merged**
